.github/ISSUE_TEMPLATE/implementation-ticket.yml

```diff
@@ -0,0 +1,66 @@
+name: 🛠️ Implementation
+description: This is an implementation ticket intended for use by the maintainers of `dbt-adapters`
+title: "[<project>] <title>"
+labels: ["user docs","refinement"]
+body:
+  - type: markdown
+    attributes:
+      value: This is an implementation ticket intended for use by the maintainers of `dbt-adapters`
+
+  - type: checkboxes
+    attributes:
+      label: Housekeeping
+      description: >
+        A couple friendly reminders:
+          1. Remove the `user docs` label if the scope of this work does not require changes to https://docs.getdbt.com/docs: no end-user interface (e.g. yml spec, CLI, error messages, etc) or functional changes.
+          2. Will this change need to be backported? Add the appropriate `backport 1.x.latest` label(s).
+          3. Is this an `enhancement`, `bug`, or `regression`? Add the appropriate label.
+      options:
+        - label: I am a maintainer of `dbt-adapters`
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Short description
+      description: |
+        Describe the scope of the ticket, a high-level implementation approach and any tradeoffs to consider.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Context
```

Comment:
I moved the context section to before the AC. I think it helps to have this in your head when reading through the AC. This is similar to our Problem / Solution setup for PRs.